maintainers/scripts/nix-diff.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p coreutils
+
+################################################################################
+# nix-diff.sh
+################################################################################
+# This script "diffs" generations of the current environment (whatever
+# environment is linked to ~/.nix-profile).  Currently, this diff is simply a
+# list of derivations that have changed between two given generations, along
+# with an indication of whether each derivation was added or removed.
+#
+# Example:
+############################################################
+# > nix-diff 90 92
+# + gnumake-4.2.1
+# + gnumake-4.2.1-doc
+# - htmldoc-1.8.29
+############################################################
+# The example shows that as of generation 92 and since generation 90,
+# gnumake-4.2.1 and gnumake-4.2.1-doc have been installed, while htmldoc-1.8.29
+# has been removed
+################################################################################
+
+usage() {
+    cat <<EOF >&2
+usage: nix-diff.sh [-h] parent [child]
```

Comment:
This relies on env-manifest.nix, I think `nix-env-diff` or something would a better name.